model/user-agent/registry.yaml

```diff
@@ -35,3 +35,83 @@ groups:
           using a user-agent for non-browser products, such as microservices with multiple names/versions inside the
           `user_agent.original`, the most significant version SHOULD be selected. In such a scenario it should align
           with `user_agent.name`
+
+  - id: registry.user_agent.os
+    type: attribute_group
+    display_name: User-agent OS Attributes
+    brief: "Describes the OS user-agent attributes."
+    attributes:
+      - id: user_agent.os.type
+        type:
+          members:
+            - id: windows
+              value: 'windows'
+              brief: "Microsoft Windows"
+              stability: experimental
+            - id: linux
+              value: 'linux'
+              brief: "Linux"
+              stability: experimental
+            - id: darwin
+              value: 'darwin'
+              brief: "Apple Darwin"
+              stability: experimental
+            - id: freebsd
+              value: 'freebsd'
+              brief: "FreeBSD"
+              stability: experimental
+            - id: netbsd
+              value: 'netbsd'
+              brief: "NetBSD"
+              stability: experimental
+            - id: openbsd
+              value: 'openbsd'
+              brief: "OpenBSD"
+              stability: experimental
+            - id: dragonflybsd
+              value: 'dragonflybsd'
+              brief: "DragonFly BSD"
+              stability: experimental
+            - id: hpux
+              value: 'hpux'
+              brief: "HP-UX (Hewlett Packard Unix)"
+              stability: experimental
+            - id: aix
+              value: 'aix'
+              brief: "AIX (Advanced Interactive eXecutive)"
+              stability: experimental
+            - id: solaris
+              value: 'solaris'
+              brief: "SunOS, Oracle Solaris"
+              stability: experimental
+            - id: z_os
+              value: 'z_os'
+              brief: "IBM z/OS"
+              stability: experimental
+        brief: >
+          The operating system type.
+        stability: experimental
+      - id: user_agent.os.description
+        type: string
+        stability: experimental
+        brief: >
+          Human readable (not intended to be parsed) OS version information,
+          like e.g. reported by `ver` or `lsb_release -a` commands.
+        examples: ['Microsoft Windows [Version 10.0.18363.778]', 'Ubuntu 18.04.1 LTS']
+      - id: user_agent.os.name
+        type: string
+        stability: experimental
+        brief: 'Human readable operating system name.'
+        examples: ['iOS', 'Android', 'Ubuntu']
+      - id: user_agent.os.version
+        type: string
+        stability: experimental
+        brief: >
+          The version string of the operating system as defined in
+          [Version Attributes](/docs/resource/README.md#version-attributes).
+        examples: ['14.2.1', '18.04.1']
+      - id: user_agent.os.build_id
```

Comment:
same question for this attribute, thanks

Reply:
Removing it until we have a use case https://github.com/open-telemetry/semantic-conventions/pull/1434/commits/1b46946a1d752f6e7fc157c22ec6670262eb7b11

wdyt? Thanks for the review!